packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -0,0 +1,257 @@
+import '@aws-cdk/assert/jest';
+import * as iam from '@aws-cdk/aws-iam';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as core from '@aws-cdk/core';
+import * as path from 'path';
+import * as inc from '../lib';
+import * as futils from '../lib/file-utils';
+
+// tslint:disable:object-literal-key-quotes
+/* eslint-disable quotes */
```

Comment:
I'd prefer we drop these exclusions. 

Running `eslint --ext=.ts --fix` from your terminal should autofix this across all your files. Ideally, `yarn lint --fix` should do this but I don't think it works yet.

Reply:
These are here on purpose, so that the assertions can look like this:

```ts
    expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
      "PolicyDocument": {
        "Statement": [
          {
            "Action": "s3:*",
            "Resource": {
              "Fn::GetAtt": [
                "Bucket",
                "Arn",
              ],
            },
          },
        ],
      },
    });
```

Which look more natural than single quotes, as JSON uses double quotes (also, makes it easier to copy-paste fragments between the template and the tests).